app/scripts/modules/core/src/entityTag/EntityTagEditor.tsx

```diff
@@ -123,6 +123,7 @@ export class EntityTagEditor extends React.Component<IEntityTagEditorProps, IEnt
     const entityType = ownerOption ? ownerOption.type : this.props.entityType;
 
     const entityRef: IEntityRef = this.props.entityRef || EntityRefBuilder.getBuilder(entityType)(owner);
+    if (!entityRef.application) entityRef.application = application.name;
```

Comment:
I think we usually break these into multiple lines: 
```js
if (!entityRef.application) {
  entityRef.application = application.name;
}
```

Reply:
Yes, I'd look at what the linter picked up here: https://travis-ci.org/spinnaker/deck/builds/398268296?utm_source=github_status&utm_medium=notification